manifests/params.pp

```diff
@@ -47,6 +47,8 @@
 
   $vhost_include_pattern = '*'
 
+  $modsec_audit_log_parts = 'ABIJDEFHZ'
```

Comment:
According to [the docs](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual#SecAuditLogParts) the default is `ABCFHZ`. Why did you choose this instead? It actually says that `D` isn't even implemented yet.